src/__tests__/prompt.test.ts

```diff
@@ -0,0 +1,30 @@
+import { OCR_SYSTEM_PROMPT, RECEIPT_SCAN_PROMPT, RECEIPT_VALIDATION_PROMPT } from '../const/prompt';
+
+describe('Prompt Constants', () => {
+  test('OCR_SYSTEM_PROMPT should be defined', () => {
+    expect(OCR_SYSTEM_PROMPT).toBeDefined();
```

Comment:
Right!    We need some sort of repeat 10x  and `expect 7 or greater as passed`.    Will require some creative coding.    Please try to solve.